src/new-client/src/app/lib/services/appointmentsService.ts

```diff
@@ -36,6 +43,22 @@ export const fetchSite = async (siteId: string) => {
   return userProfile?.availableSites.find(s => s.id === siteId);
 };
 
+export const fetchSiteAttributeValues = async (siteId: string) => {
+  const response = await appointmentsApi.get<SiteWithAttributes>(
+    `sites/${siteId}`,
+  );
+
+  return handleResponse(response)?.attributeValues ?? [];
+};
+
+export async function fetchAttributeDefinitions() {
+  const response = await appointmentsApi.get<AttributeDefinition[]>(
+    'attributeDefinitions',
```

Comment:
I'm just wondering if it's worth adding `cache: 'force:cache'` to this request, since we'd expect this response to change shape incredibly rarely? I don't think there's much risk in using a stale result here

Reply:
That, or setting an extremely long cache time?